src/components/NetworkAlert/NetworkAlert.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import Icon from 'semantic-ui-react/dist/commonjs/elements/Icon'
+import { Button } from '../Button/Button'
+import { Header } from '../Header/Header'
+import { Props } from './NetworkAlert.types'
+import './NetworkAlert.css'
+
+export class NetworkAlert extends React.PureComponent<Props> {
+  static defaultProps = {
+    i18n: {
+      title:
+        'Your wallet is connected to a partially supported network (Polygon)',
+      content: (
+        <span>
+          Switch to Ethereum Mainnet if you want to use all the features of this
+          app. <a href="">Learn more</a>
```

Comment:
href tbd?

Reply:
I've used the default values as placeholders for storybook. The real values will be set in the `decentraland-dapps` repository later. Do you think we should set it here as well?

Reply:
got it!